prow/plank/controller.go

```diff
@@ -535,27 +538,27 @@ func (c *Controller) syncTriggeredJob(pj prowapi.ProwJob, pm map[string]coreapi.
 
 // TODO: No need to return the pod name since we already have the
```

Comment:
In case we have an error on the ProwJob update, we loose the time. I don't think it matters too much, but taking the creation time sounds more accurate, since it can be re-discovered on retries.

Reply:
Change to use `now()`.